addons/index.md

```diff
@@ -7,16 +7,52 @@ layout: documentation
 # Add-ons
 
 All add-ons for openHAB 2 are part of the distribution.
-This includes all new 2.0 bindings as well as all 1.x add-ons that were reported to be compatible.
+This includes all new 2.0 bindings as well as all 1.x add-ons
```

Comment:
Due to the linting warnings, which told me to not extend lines above 80 characters. I wanted to talk about that anyway on a later point.

Reply:
Yeah - something is not right with your linter.  The line limit is something like 230 characters.  I forget exactly, but it is a lot longer than 80.

Reply:
It's [240](https://github.com/openhab/openhab-docs/blob/gh-pages/package.json#L74).
Rationale: 80 is too short and unsuited to the modern world. Other common line length restrictions are 120 and 160. 
Combined with the rule to have one sentence per line a maximum line length is great to assure that. A reasonable/understandable sentence shouldn't be much longer that 160 characters. However due to inserted links, a line can quickly become longer than the sentence would be by its own.

Long story short, I chose 240 because (1) it's more than long enough to fit all sentences but (2) will expose multiple concatenated sentences and on top of all (3) it's double the line length of GitHubs file and diff view 😉

Reply:
+1
Listing will always involve some human decisions.  I find that 240 works just fine.  When a sentence gets longer than that, I start having parser faults as I try to keep everything in my head.  But, as you say, if there are a lot of links, then lines longer than 240 are just fine.  I like it.

Reply:
I know that it is 240 usually and i can't tell you why it tells me to use 80 now.
I just did the npm install on the repo with the latest package.json at that time and nothing more.

### So long story short for me

I am stopping linter related edits for now completely and everywhere.
I am not willed to investigate another 5 hours in "why is this crap not working correct on my setup".
That's a huge waste of time.
Time which I will invest in other topics now.

Sorry for the hard words, but i am really pissed of off following the warnings in atom editor (which i installed only for this purpose) and then getting told that somethings wrong afterwards.
That's not the first time and i am sure it won't be the last.

If i set up and install everything like @ThomDietrich mentioned it in several places and threads
and it still won't work for me, than i have to give up for now.
Voluntary work is a good thing for me and i like helping out and improving docs or whatever, 
but at the end i want to shut down my machine with a good feeling and not with anger about that stupid linter, which again doesn't work correct.

Reply:
I know that feeling...

Reply:
@Confectrician Uff! Sorry you are having such a rough time with the linter settings. I totally get your point and agree that the linter should just work as a way of helping rather than holding you back.
I found that it's not always clear if the editor actually uses the custom settings or is annoying you with the default settings. There should be an option to explicitly ask to use project settings or stay silent.
Maybe something we should mention to the Atom plugin developer. What do you guys think?

Anyhow, you seem to be angered right now so I'll not bother you with the linter settings now. When you are ready to, how about a team discussion in which we can shortly discuss all setup steps and checks leading to a working linter experience. The result could then finally be stored in CONTRIBUTING or styleguide.md

Nicht unterkriegen lassen! 😃